src/dk/aau/cs/io/TapnLegacyXmlLoader.java

```diff
@@ -318,7 +319,7 @@ private Arc parseAndAddTimedInhibitorArc(String idInput, boolean taggedArc,
 			firstInhibitorIntervalWarning = false;
 		}
 		
-		TimedInhibitorArc inhibArc = new TimedInhibitorArc(place, transition, interval);
+		TimedInhibitorArc inhibArc = new TimedInhibitorArc(place, transition, interval,null);
```

Comment:
We should avoid passing null, should the constructor be overloaded or given default element?